docker/Dockerfile

```diff
@@ -64,6 +64,8 @@ COPY --from=builder /risingwave/bin/jeprof /usr/local/bin/jeprof
 ENV PLAYGROUND_PROFILE docker-playground
 # Set default dashboard UI to local path instead of github proxy
 ENV RW_DASHBOARD_UI_PATH /risingwave/ui
+# Set default connector libs path
+ENV CONNECTOR_LIBS_PATH /risingwave/bin/connector-node/libs
```

Comment:
Add default CONNECTOR_LIBS_PATH to the Dockerfile.

Reply:
Do I miss any other Dockerfiles? cc @fuyufjh @BugenZhao

Reply:
AFAIK this is the only one